data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,162 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+
+# ahca_polls ----------------------------------------------------------------------
+ahca_polls <- read_csv("data-raw/ahca-polls/ahca_polls.csv") %>% 
+  clean_names() %>% 
+  mutate(
+    start = as.Date(start, "%m/%d/%y"),
+    end = as.Date(end, "%m/%d/%y"),
+    pollster = as.factor(pollster)
+  )
+devtools::use_data(ahca_polls, overwrite = TRUE)
+
+
+# bachelorette ---------------------------------------------------------------------
+bachelorette <- read_csv("data-raw/bachelorette/bachelorette.csv") %>%
+  clean_names() %>%
+  mutate(
+    season = as.factor(season),
+    elimination_1 = as.factor(elimination_1),
+    elimination_2 = as.factor(elimination_2),
+    elimination_3 = as.factor(elimination_3),
+    elimination_4 = as.factor(elimination_4),
+    elimination_5 = as.factor(elimination_5),
+    elimination_6 = as.factor(elimination_6),
+    elimination_7 = as.factor(elimination_7),
+    elimination_8 = as.factor(elimination_8),
+    elimination_9 = as.factor(elimination_9),
+    elimination_10 = as.factor(elimination_10),
+    dates_1 = as.factor(dates_1),
+    dates_2 = as.factor(dates_2),
+    dates_3 = as.factor(dates_3),
+    dates_4 = as.factor(dates_4),
+    dates_5 = as.factor(dates_5),
+    dates_6 = as.factor(dates_6),
+    dates_7 = as.factor(dates_7),
+    dates_8 = as.factor(dates_8),
+    dates_9 = as.factor(dates_9),
+    dates_10 = as.factor(dates_10)
+  ) %>%
+  filter(
+    season != "SEASON"
+  )
+devtools::use_data(bachelorette, overwrite = TRUE)
```

Comment:
Could you go thru all datasets and replace all instances of `<NA>` with `NA`? The former looks cluttered. See code example below.

Also note when repetitively applying the same function, in this case `as.factor()`, to very similarly named variables, in this case `elimination_X` and `dates_X`, it's worth seeing if using `mutate_at` along with a `?dplyr::select_helpers` function like `starts_with` can help minimize redundancy:
```
bachelorette <- read_csv("data-raw/bachelorette/bachelorette.csv") %>%
  clean_names() %>% 
  mutate_at(vars(starts_with("elimination")), as.factor) %>%
  mutate_at(vars(starts_with("elimination")), funs(ifelse(. == "<NA>", NA, .))) %>%
  mutate_at(vars(starts_with("dates")), as.factor) %>%
  mutate_at(vars(starts_with("dates")), funs(ifelse(. == "<NA>", NA, .))) %>% 
  filter(season != "SEASON")
```

Reply:
Also, make `season` an integer